resources/js/i18n/en/rules.js

```diff
@@ -70,33 +70,33 @@ export default {
   boardRulesTitle: 'Board',
   field: {
     title: 'Field',
-    description: 'The place to which your characters are summoned. Attacking side (AF) defending side (DF) are three rows each. This is a total of 6 spots.',
-    point1: 'One spot can only have one character on it.',
-    point2: 'You can neither move, nor send the characters to the graveyard without an explicit basic ability or an effect.',
+    description: 'This is where characters are summoned to. It consists of an Attack Field (AF) and Defense Field (DF), with each field containing 3 spots. In total there are 6 spots on the field.',
+    point1: 'Each spot can be occupied by only 1 character.',
+    point2: 'Once summoned, characters cannot be move around or leave the field unless explicitly stated.',
     imageAlt: 'Picture of the board of a player\'s side.',
   },
   graveyard: {
-    title: 'Graveyard',
-    description1: 'The place your cards that are destroyed or discarded go.',
-    description2: 'All cards here are face-up, and either player can check each others\' at any time.',
+    title: 'Discard Pile',
+    description1: 'This is where discarded and destroyed cards go.',
+    description2: 'The discard pile is shown face-up; both players can check each others\' discard pile at any time.',
   },
   deck: {
     description1: 'Your deck.',
-    description2: 'Its contents are face-down, and neither player is allowed to look at them.',
+    description2: 'The deck is placed face-down; neither player is allowed to look at its contents.',
   },
   rowsAndColumns: {
-    title: 'Rows and columns',
-    description: 'Lines of fields are called rows and columns.',
-    point1_line1: 'Looking at the image above, a lines going from left to right are rows (AKA. "orders"), and lines going from bottom to top are colums.',
-    point1_line2: 'For example, in the above figure, the upper right "Ruler / Jeanne d\'Arc" character located in the AF row of your right column.',
-    point1_line3: '(From your opponent\'s point of view, it\'s on the left column.)',
-    point2: 'A spot on the field can be referenced as "a field" in card texts.',
-    point3: 'The DF is also known as the back row, and AF as the front row.',
+    title: 'Orders and Columns',
```

Comment:
Please make "Row" the official term and "Order" as a secondary AKA term. Same applies to other entries on this page.
Does Github allow you to edit files already submitted for a PR to make additional changes?